dotnet/src/webdriver/Chromium/ChromiumDriver.cs

```diff
@@ -469,6 +471,18 @@ protected override void Dispose(bool disposing)
                     this.devToolsSession.Dispose();
                     this.devToolsSession = null;
                 }
+
+                if (this.SessionId is not null)
+                {
+                    this.Execute(DriverCommand.Quit, null);
```

Comment:
Let's return back try/catch pattern as it was before. Then we can improve later.